### PR TITLE
[FICTIUM-005] Add Rspec integration

### PR DESCRIPTION
**Pull request by holywyvern**
## Summary

This PR adds the RSpec DSL used by the gem to auto generate documentation.

It requires some minor adjustments, but it allows you to generate quick documentation based on . your RSpec files, without touching most of it.

### CODE REVIEW

File: lib/fictium/configurations/configuration.rb [hunk 1]
```diff
@@ -1,10 +1,62 @@
 module Fictium
   class Configuration
-    attr_accessor :exporters
+    VOWEL = /[aeiou]/i.freeze
+    VERB_TERMINATOR = /[hstv]/i.freeze
```
Comment (mtejedorwolox) [inline lib/fictium/configurations/configuration.rb]: unused

File: lib/fictium/poros/resource.rb [hunk 2]
```diff
@@ -1,15 +1,27 @@
 module Fictium
   class Resource < Fictium::Model
+    REVERSE_CONTROLLER = 'Controller'.reverse.freeze
```
Comment (mtejedorwolox) [inline lib/fictium/poros/resource.rb]:
```suggestion
    CONTROLLER = /Controller$/.freeze
```

File: lib/fictium/poros/resource.rb [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
       @actions = []
+      @tags = []
     end
 
     def create_action
       Fictium::Action.new(self).tap { |action| actions << action }
     end
+
+    def name_attributes(controller_name)
+      resource_path = controller_name.reverse.sub(REVERSE_CONTROLLER, '').reverse.underscore
```
Comment (mtejedorwolox) [inline lib/fictium/poros/resource.rb]:
```suggestion
      resource_path = controller_name.sub(CONTROLLER, '').underscore
```

File: lib/fictium/configurations/configuration.rb [hunk 4]
```diff
@@ -1,10 +1,62 @@
 module Fictium
   class Configuration
-    attr_accessor :exporters
+    VOWEL = /[aeiou]/i.freeze
```
Comment (lucasVoboril) [inline lib/fictium/configurations/configuration.rb]: If you are only create this constant for this file, you should add this line
```rb
VOWEL = /[aeiou]/i.freeze
private_constant :VOWEL
```
Also, delete VERB_TERMINATOR unused constant
[private_constant](http://www.chrisrolle.com/en/blog/private-constants-in-ruby)

**Pull request merged**